python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceText2ImageDiffusorPromptSchema.ts

```diff
@@ -0,0 +1,100 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const HuggingFaceText2ImageDiffusorPromptSchema: PromptSchema = {
+  // See /Users/ryanholinshead/Projects/aiconfig/extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/text_2_image.py
+  // for supported settings.
+  // See https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/text2img#diffusers.StableDiffusionPipeline.__call__
+  // for descriptions.
+
+  // The following supported properties cannot be cleanly modeled in the schema and must be implemented programatically:
+  // generator, latents, prompt_embeds, negative_prompt_embeds, callback
```

Comment:
Do you have thoughts on how this would be modeled? Will the model parser need to handle them?

Reply:
This is because they're all torch objects right?

What we do now is first filter for generator params, and then we do completion params. We could do something similar where we have two sets of input schemas for images

Reply:
I would need to spend more time looking into it to know, but I assume they are serialized as python objects in the config? It's just hard to map generic python objects to schema-supported types